src/main/java/com/questhelper/QuestHelperConfig.java

```diff
@@ -112,11 +164,33 @@ default QuestOrdering orderListBy()
 	@ConfigItem(
 		keyName = "filterListBy",
 		name = "Filter",
-		description = "Configures what to filter out in the quest list",
+		description = "Configures what to filter in the quest list",
 		position = 1
 	)
 	default QuestFilter filterListBy()
 	{
-		return QuestFilter.HIDE_DONE;
+		return QuestFilter.SHOW_ALL;
+	}
+
+	@ConfigItem(
```

Comment:
I'm not sure about having difficulty filtering as is currently implemented, it feels a bit confusing having to select two different filters in combination to get it working.

Looking at it now, I reckon having the filters separated to tick boxes would work better. This'd allow users to stack filters which could be nice as well.

I'd say sections would be 'Hide done', 'Hide quests', 'Hide miniquests', 'Hide can't do' 'Hide P2P', 'Hide F2P', then the current dropdown for difficulty filtering which has the additional 'All'.

WDYT?

Reply:
I'm actually not a fan of separating everything into checkboxes. 
Mainly because how often will you be stacking filters like that. I think the filter that are present are pretty much what's going to be searched for.
As well, I think it would just be too many options for the user to see at once. The only quests we really need to filter out are those that are completed (or not). The rest of the quests fall into one of the filter categories nicely.

I can add the 'All' difficulty option though, that's pretty easy.